pkgs/by-name/hc/hclust2/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, fetchFromGitHub
+, nix-update-script
+, python3Packages
+, python3
+, phylophlan
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "hclust2";
+  version = "1.0.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "SegataLab";
+    repo = "hclust2";
+    rev = version;
+    hash = "sha256-xdS36Sfxg4bz5ztRbCdD3uq4Dx50E8n501ScMArjwso=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+  ];
+
+  propagatedBuildInputs = with python3Packages; [
+    numpy
+    scipy
+    pandas
+    matplotlib
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = with lib; {
+    description = "Hclust2 is a handy tool for plotting heat-maps with several useful options to produce high quality figures that can be used in publication.";
```

```suggestion
    description = "Tool for plotting heat-maps with several useful options to produce high quality figures that can be used in publication";
```

see https://nixos.org/manual/nixpkgs/stable/#var-meta-description